testing/citest/spinnaker_testing/spinnaker.py

```diff
@@ -88,6 +88,14 @@ def name_value_to_dict(content):
     result[match.group(1)] = match.group(2).strip()
   return result
 
+def construct_base_url(host, port, protocol=None):
+  """Constructs a base_url from host, port, and an optional protocol.
+  """
+  return '{protocol}{host}:{port}'.format(
```

Comment:
should be something like "http" or "https", so you should be adding in the "//" separator between the URL's protocol and host.